src/main/java/sirius/biz/process/ErrorContext.java

```diff
@@ -147,6 +147,7 @@ public String getContextAsString() {
     private void logException(UnaryOperator<String> failureDescription, HandledException exception) {
         if (TaskContext.get().getAdapter() instanceof ProcessContext) {
```

Comment:
Optional: bind getAdapter to a variable and use instanceof pattern matching to remove cast below